### PR TITLE
Import strings from android-l10n

### PR DESCRIPTION
**Pull request by Pike**
State: mozilla-l10n/android-l10n@413e05dba427ef8cbf9e5272ae5cc3b3ccdf91d3

I wonder what's going to become of `sync_timed_out` in the app repo. I'm hesitent to take action on the l10n side, but given how long this has been commented out, is that string just dead?

CC @Delphine to also see the replies.

### CODE REVIEW

**Comment by eliserichards** [comment]
The tentative plan is to remove the sync timeout from our end. Sorry for the ambiguity there! @Pike @Delphine

**Comment by ddurst** [comment]
@Pike -- if we determine that we're removing the timeout, then we won't need the string; that's TBD over the next week or so.

But just to make sure: the vietnamese XML strings are already added to Pontoon and coming in in this patch from there?

**Comment by Pike** [comment]
Yes, they're working on https://pontoon.mozilla.org/vi/android-l10n/tags/ now. Delphine connected the contributor to the team there.

**Comment by ddurst** [comment]
OK, so we'll go ahead and remove the timeout, and those strings will then be gone. The rest of this will exist on the pontoon side.

**Comment by Pike** [comment]
I don't think it's necessary to wait on that roundtrip for this PR.

Also, how do you guys handle the `question` label? Should I have removed that, or are there open questions left, or is that on the person that asked to say "yeah, got a sufficient answer" ?

**Comment by ddurst** [comment]
Eh, no idea. I meant it as a question about whether or not we're removing `timeout` anyway, so I'll remove it. :)

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -252,5 +252,5 @@
 
     <!-- This is the placeholder text for an empty or null username. -->
     <string name="no_username">(no existe usuario)</string>
-    <!--<string name="sync_timed_out">La sincronización ha expirado</string>-->
+    <string name="sync_timed_out">La sincronización ha expirado</string>
```
Comment (jhugman) [inline app/src/main/res/values-es/strings.xml]: Is this being removed, or uncommented?

Comment (Pike) [reply app/src/main/res/values-es/strings.xml]: Uncommented. We'll likely want a force-sync on the pontoon side once the string is actually removed. CC @Delphine for that.

**Comment by eliserichards** [comment]
Cleaned up and merged in #893 😄

**Pull request closed without merging**